core/embed/rust/src/ui/model_mercury/layout.rs

```diff
@@ -2150,9 +2110,29 @@ pub static mp_module_trezorui2: Module = obj_module! {
     ///     """Show single-line text in the middle of the screen."""
     Qstr::MP_QSTR_show_wait_text => obj_fn_1!(new_show_wait_text).as_obj(),
 
-    /// def flow_get_address() -> LayoutObj[UiResult]:
+    /// def flow_get_address(
```

Comment:
exposing the functions as `flow_xyz` is a good idea for now. I will use it also in flows I'm working on.

Reply:
eventually we probably want them in separate python module but for now just place them here